lib/tasks/pictograms.rake

```diff
@@ -135,7 +142,7 @@ namespace :pictograms do
 
     Dir.entries(tmp_load_folder).reject {|f| File.directory? f}.each do |entry|
       img = File.new File.join(tmp_load_folder, entry)
-      Pictogram.create! image: img
+      Pictogram.create! image: img, locale: 'pt-BR'
```

Comment:
Given https://github.com/marcelorcorrea/falae/pull/54, we need to update locale value to 'pt'